core/src/main/resources/db/changelog/changesets/2025_07_28_custom_kit_eligibility.yaml

```
@@ -0,0 +1,14 @@
+databaseChangeLog:
+  - changeSet:
+      id: "custom_kit_eligibility"
+      author: connorlbark
+      changes:
+        - addColumn:
+            tableName: study_environment_config
+            columns:
+              - column:
+                  name: kit_eligibility_rule
+                  type: text
+                  constraints:
+                      nullable: true
```

nullable is the default, no need to call it out explicitly
```suggestion
                  type: text
```